extensions/amp-ima-video/0.1/amp-ima-video.js

```diff
@@ -244,8 +244,8 @@ class AmpImaVideo extends AMP.BaseElement {
    * @private
    */
   sendCommand_(command, opt_args) {
-    if (this.iframe_ && this.iframe_.contentWindow) {
-      this.playerReadyPromise_.then(() => {
+    this.playerReadyPromise_.then(() => {
```

Comment:
`playerReadyPromise_` may be null.

Reply:
Is it possible for any of these methods to be called before `layoutCallback` which sets the promise? `unlayoutCallback` doesn't reset it to null, it sets it to another promise. Regardless, added a check here to be safe.